### ISSUE TITLE
Create list of experts

### ISSUE THREAD

**Issue by natdudley** [WikiNewZealand/pacific#16]
expert page needs list of experts

**Comment by natdudley** [comment]
This is in progress now.

@ngapera can you confirm if we have permission yet to include the Community Research website? I vaguely recall you said that they hadn't officially launched yet.

**Comment by Ngapera** [comment]
> This is in progress now.
> 
> @Ngapera can you confirm if we have permission yet to include the Community Research website? I vaguely recall you said that they hadn't officially launched yet.

Yes we have permission from them. I'm still waiting on permission for another link to Pacific digital tools, but I will add that later

**Comment by natdudley** [comment]
Fab. Thanks!

If/when you get permission to add more people (i've highlighted in red on spreadsheet ones we can't add yet) could you please open another issue for them to let us know?

In the meantime, I'm going to assign the PR to merge branch to add the existing ones we have permissions for and close this issue.
